backend.py

```diff
@@ -73,8 +73,9 @@ def walk(self, distance, state, direction=None):
 
                 # Robot walks to next coordinates.
                 self.coordinates = next_coordinates
-                # Check hole on next coordinates.
-                self.check_hole(state)
+                # If robot falls into hole, he becomes inactive.
+                if self.inactive:
```

Comment:
Akorát ti tady chybí ten pád do díry. Bez toho robotovi život neubyde :)

Reply:
Já právě nevím jak to udělat protože když to tam dám tak ta hra opět zase spadne :( Ještě zkusím.

Reply:
Tak jsem si to zkusila. Do `move` i `walk` ten pád do díry patří. Další problém se pak objeví ve frontendu. Tam je problém s podmínkou na řádku 47. To je podmínka, která má zaručit, že se vykreslí pouze aktivní roboti na mapě. Momentálně je napsaná špatně. Zkus se na to mrknout, když ti napovím ;)

Reply:
@JureckovaK napověz 😂

Reply:
prosím tě tenhle PR zavírám, nějak se mi pokazily větve a uplně se mi nahrály jiné commity, udělám to znovu, už jsem to vyřešila i ve frontendu, tak tě pak poprosím o review.